test-configs.yaml

```diff
@@ -179,7 +180,57 @@ test_plans:
         kms_atomic
         kms_flip_event_leak
         kms_prop_blob
-        kms_setmode kms_vblank
+        kms_setmode
+        kms_vblank
+
+  kms-tegra:
+    rootfs: debian_buster-igt_ramdisk
+    pattern: 'igt/{category}-{method}-{protocol}-{rootfs}-igt.jinja2'
+    params:
+      drm_driver: "tegra"
+      tests: >
+        core_auth
+        core_getclient
+        core_getstats
+        core_getversion
+        core_setmaster_vs_auth
+        drm_read
+        kms_addfb_basic
+        kms_atomic
+        kms_flip_event_leak
+        kms_prop_blob
+        kms_setmode
+        kms_vblank
+
+  kms-exynos:
+    rootfs: debian_buster-igt_ramdisk
+    pattern: 'igt/{category}-{method}-{protocol}-{rootfs}-igt.jinja2'
+    params:
+      drm_driver: "exynos"
+      tests: >
```

Comment:
If it's the same list of test cases for all the kms variants, you can create an anchor on the first one and then references for the other ones.  This avoids having to maintain multiple identical lists of test cases.